dashboard.php

```diff
@@ -35,39 +37,28 @@
                         </div>
                     </div>
                     <ul class="jobs-listing">
+
+                    <?php
+                    $sql_jobs = "SELECT jobs.id, jobs.title, jobs.salary, jobs.location, jobs.date_posted, jobs.description, users.phone, users.company_name, users.company_site FROM jobs 
+                    LEFT JOIN users ON jobs.user_id = users.id ORDER BY date_posted";
```

Comment:
SQL queries on a single line is  ugly. You can do something like this:
```suggestion
                    $sql_jobs = "SELECT jobs.id, jobs.title, jobs.salary, jobs.location, jobs.date_posted, jobs.description,
                                 users.phone, users.company_name, users.company_site
                                 FROM jobs 
                                 LEFT JOIN users ON jobs.user_id = users.id
                                 ORDER BY date_posted";
```